root/tree/dataframe/RDataFrameOpenDataBenchmarks.cxx

```diff
@@ -596,7 +596,7 @@ void benchmark8_compiled(unsigned int nthreads)
                       {"nMuon", "nElectron"})
               .Define("AdditionalLepton_idx", benchmark8_additional_lepton_idx,
                       {"Lepton_pt", "Lepton_eta", "Lepton_phi", "Lepton_mass", "Lepton_charge", "Lepton_flavour"})
-              .Filter([](int idx) { return idx != -999; }, {"AdditionalLepton_idx"}, "No valid lepton pair found.")
+              .Filter([](unsigned int idx) { return idx != 99999; }, {"AdditionalLepton_idx"}, "No valid lepton pair found.")
```

Comment:
Would it make sense for this magic constant to become a preprocessor macro?

Reply:
Hmm, I don't think it would make a difference. The value `99999` is used to indicate that we did not find a better `AdditionalLepton_idx`. If a better index is found then it is updated (hence is not constant).

Reply:
Yet, we changed a magic constant `-999` to another magic constant `99999` in couple of places ;)

Reply:
Also in the other places it was changed for the same reason. Alternative is to have a macro which is equal to `99999`, and initialize these helping variables to the macro. But does it make a difference?

Reply:
I think it does as the next time one thing will change in one place, right?

Reply:
I agree. By this token, there are also other tasks in this benchmark which can benefit from macros. To be updated very soon.

Reply:
Hi, I don't have a particular opinion on leaving a placeholder value vs adding a variable called PLACEHOLDER_VALUE, but if you go the latter way please use a constexpr static variable rather than a preprocessor macro (it's a bit more type-safe) and remember to keep these benchmarks and the code at https://github.com/root-project/opendata-benchmarks in sync.

Reply:
After taking a look at the other queries, there seems to be no need for placeholders there.

Suggested changes are applied.